railgun/internal/mgrutils/ingressclass.go

```diff
@@ -0,0 +1,6 @@
+package mgrutils
+
+import "github.com/kong/kubernetes-ingress-controller/pkg/annotations"
+
+// IngressClass is the current "kubernetes.io/ingress.class" configured for controllers
```

Comment:
`"kubernetes.io/ingress.class"` is misleading because this annotation is deprecated on Ingress resources in favor of `spec.IngressClassName`.

Reply:
```suggestion
// IngressClass is the current ingress class name configured for controllers
```